### ISSUE TITLE
Paper Archive - Structured Filenames tab

### ISSUE THREAD

**Issue by isedwards** [openclimateinitiative/climsoft-app#3]
The <kbd>Structured Filenames</kbd> tab lists image files from a directory on the users local machine that are in a structured format of the form **StdID-FormID-YYMMDDHH**. The user can select which images they want to upload.

In the case of the old software the files were copied to a different location - either a local drive or a network drive. This is shown at the bottom of the window: `Images Archived at:   C:\PaperArchive\Images`

For this new implementation the files should be uploaded to the server using the API (more details to follow).

We don't necessarily need the large box on the right hand side, this just shows an warnings or errors. In the image below the Station ID (StdID) is 67774010, the Form ID is 61001 and then the year, month and day are given. No hour (HH) is given because the data is in the image is for an entire day.

![structured_filenames](https://user-images.githubusercontent.com/2097922/154305928-1cff38b9-882d-49eb-8d3b-72e897acef34.png)

**Comment by stteem** [comment]
For this feature, the HTML file upload API opens up the local drive for multiple file upload. I'm assuming the files will have been preformatted/stored according to the structured format. It could look like this, but without a modal.

![upload](https://user-images.githubusercontent.com/30989492/155522278-25ad0ab2-9cde-4473-bf6a-9dac4dcfc393.PNG)

**Comment by isedwards** [comment]
Yes - for this tab multiple files can be uploaded by they must follow the expected filename structure otherwise there will be an error.

The unstructured filenames tab will allow the images to have any name, but they will need to be uploaded one at a time with the user entering the correct metadata at the same time that the image is uploaded.
